opennft/opennft.py

```diff
@@ -119,7 +107,12 @@ class OpenNFT(QWidget):
     def initUdpSender(self):
         if not config.USE_UDP_FEEDBACK:
             return
-        self.udpSender = Udp(IP=config.UDP_FEEDBACK_IP,port=config.UDP_FEEDBACK_PORT,control_signal=config.UDP_FEEDBACK_CONTROLCHAR,encoding='UTF-8')
+        self.udpSender = Udp(
+            IP=config.UDP_FEEDBACK_IP,
```

Comment:
@tiborauer Hi Tibor,

It seems the following attributes are missing in `config`:
* USE_UDP_FEEDBACK
* UDP_FEEDBACK_IP
* UDP_FEEDBACK_PORT
* UDP_FEEDBACK_CONTROLCHAR

This code will cause `AttributeError` exception.

Reply:
Also this code does nothing:

```python
    def finalizeTcpReceiver(self):
        self.TcpReceiver
```

Reply:
You are right! The `initTcpReceiver` and `finalizeTcpReceiver` methods are placeholders to implement [DICOM over TCP](https://github.com/tiborauer/pynf/blob/master/example_dicomexport.py)

Reply:
I have not experienced any `AttributeError`. These config attributes are set in `OpenNFT.actualize()`:
https://github.com/OpenNFT/OpenNFT/blob/691034fae8bec61179704d8f355bf1fbdbaaf2a8/opennft/opennft.py#L1600-L1607

Reply:
Okay but I think it is not a good idea. Anyway I think we have to rewrite configuration with yaml-based config file instead of python-module.